api/google-manager/services/google-manager.js

```diff
@@ -128,8 +155,20 @@ if (!isDevEnv()) {
   const privateKey = googleKey.private_key;
   const groupID = process.env.DEVLAUNCHERS_GOOGLE_DIRECTORY_GROUP_ID;
   const serverBaseURL = process.env.URL;
+  const auditFreq = process.env.AUDIT_FREQ_MINUTES;
+
+  const manager = new GoogleManager(email, privateKey, groupID, serverBaseURL, auditFreq);
+  const cronTime = `0 */${auditFreq} * * * *`;
+  const job = new CronJob(cronTime, function () {
+    manager.listAuditReports('all', 'meet', 'call_ended').then(reports => {
+      reports.items.forEach(r => {
+        strapi.log.info('Report for Google Meet', r.events[0].parameters);
+      });
+    });
+  });
+  job.start();
 
-  module.exports = new GoogleManager(email, privateKey, groupID, serverBaseURL);
+  module.exports = manager;
```

Comment:
What is exports?

Reply:
So this service can be used in different modules.